apps/backend/src/models/device.py

```
@@ -21,16 +21,14 @@ class Device(Base):
 
     # Basic device information
     hostname = Column(String(255), unique=True, nullable=False, index=True)
-    ip_address = Column(INET, nullable=True, index=True)  # Optional - SSH config handles this
-    ssh_port = Column(Integer, nullable=True)  # Optional - SSH config handles this
-    ssh_username = Column(String(100), nullable=True)  # Optional - SSH config handles this
-
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)